FlowCryptTests/Functionallity/WKDURLs/ZBase32EncodingTests.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  ZBase32EncodingTests.swift
+//  FlowCryptTests
+//
+//  Created by Yevhen Kyivskyi on 17.05.2021.
+//  Copyright © 2021 FlowCrypt Limited. All rights reserved.
+//
+
+import XCTest
+
+class ZBase32EncodingTests: XCTestCase {
+
+    func testStringEncoding() throws {
+        let inputString = "example@email.com"
+        let encodedString = "cihgn5mopt1wy3mpcfwsamudp7so"
+
+        XCTAssert(
+            String(decoding: inputString.data().zBase32EncodedBytes(), as: Unicode.UTF8.self) == encodedString
+        )
+    }
+    
+    func testEncryptedStringEncoding() throws {
```

Comment:
Sha1 doesn't encrypt the data, it hashes it. So this would be testHashedStringEncoding